clients/js/src/algorand.ts

```diff
@@ -28,19 +30,19 @@ export async function execute_algorand(
     throw Error(`No ${network} rpc defined for Algorand`);
   }
 
-  const contracts = CONTRACTS[network][chainName];
+  const coreContract = contracts.coreBridge(network, chain);
```

Comment:
I think you'd want the `.get` here if you want to later check if it is set, similar to nftBridge one below

Reply:
In this case it is a compile time error if there is no contract for the network and chain.  But, I agree it should be consistent so as to not be confusing.  Will change.